checkout/frames-android
# feature/PIMOB-2217_Integrate_slack_message

## Issue

PIMOB-2217

## Proposed changes

Add a Slack integration depends on the build result.
![Screenshot 2023-11-14 at 17 38 45](https://github.com/checkout/frames-android/assets/123394131/89340e0b-071f-4c32-976f-9c0ff3b8ac63)


## Test Steps
- Each Slack msg will display a link for the github action page
- Each Slack msg will display a link for the github PR
- Each Slack msg will display the author of the PR
- Make expected failed build PR, it should send a Slack msg to #android-builds and says where it fails.
- Make expected successful build PR, it should send a Slack msg to #android-builds and says where it succeeded.


## Checklist


## Further comments

_If this is a relatively large or complex change, kick off the discussion by explaining why you choose the solution you did and what alternatives you considered, etc..._
